examples/platform/silabs/efr32/BaseApplication.cpp

```diff
@@ -517,6 +543,75 @@ void BaseApplication::StopStatusLEDTimer()
     }
 }
 
+#ifdef EMBER_AF_PLUGIN_IDENTIFY_SERVER
+void BaseApplication::OnIdentifyStart(Identify * identify)
+{
+    ChipLogProgress(Zcl, "onIdentifyStart");
+
+#if CHIP_DEVICE_CONFIG_ENABLE_SED == 1
+    StartStatusLEDTimer();
+#endif
+}
+
+void BaseApplication::OnIdentifyStop(Identify * identify)
+{
+    ChipLogProgress(Zcl, "onIdentifyStop");
+
+#if CHIP_DEVICE_CONFIG_ENABLE_SED == 1
+    StopStatusLEDTimer();
+#endif
+}
+
+void BaseApplication::OnTriggerIdentifyEffectCompleted(chip::System::Layer * systemLayer, void * appState)
+{
+    ChipLogProgress(Zcl, "Trigger Identify Complete");
+    sIdentifyEffect = Clusters::Identify::EffectIdentifierEnum::kStopEffect;
+
+#if CHIP_DEVICE_CONFIG_ENABLE_SED == 1
+    StopStatusLEDTimer();
+#endif
+}
+
```

Comment:
the variant part ?

Reply:
The only variant that exist currently is default. Which is the normal effect done here